doc/source/conf.py

```diff
@@ -288,7 +288,6 @@ def render_svg_logo(path):
         "csat",
     ],
     "navigation_depth": 4,
-    "analytics": {"google_analytics_id": "UA-110413294-1"},
```

Comment:
n00b question, why we are removing google analytics here?

Reply:
that will be injected by [Google Tag manager](https://developers.google.com/tag-platform/tag-manager). It is to manage all tags in one place for all websites we manage. With Google tag manager we will also inject cookiebot which will make sure Google analytics only runs when user gives permissions (where needed for compliance reasons)